pkg/cloudfoundry/ManifestUtils.go

```diff
@@ -0,0 +1,226 @@
+package cloudfoundry
+
+import (
+	"fmt"
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"reflect"
+
+	"github.com/SAP/jenkins-library/pkg/log"
+)
+
+const propApplications = "applications"
+const propBuildpacks = "buildpacks"
+const propBuildpack = "buildpack"
+
+// Manifest ...
+type Manifest struct {
+	self     map[string]interface{}
+	modified bool
+	name     string
+}
+
+var m Manifest
+
+var _readFile = ioutil.ReadFile
```

Comment:
Yes, that is not thread save. But I don't use `Parallel` in the tests.
The benefit it that I'm decoupled from the real file system. That is a benefit since otherwise the test results might be affected by what is available on the local file system. E.g. we might have a manifest file located there (maybe from earlier tests) which is not expected to be present ...